pagopa/pn-frontend-e2e
# PN-10236-10237-10238-10239-10240-10241

## Short description

Created new tests: 10236 - 10237 - 10238 - 10239 - 10240 - 10241 - 10259

## List of changes proposed in this pull request

- Created new file with notification cancellation tests. The notification is created and when it reaches a certain state is cancelled
- New general step to verify the notification state in the list of the main page (notification serached by position) in 2 minutes span
- Added some others steps where needed
- PN - 10259: added late, it checks if the destinatario see the notication cancellation

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@annullamentoNotifica"` and verify if the tests works fine.

Run on AWS: https://eu-south-1.console.aws.amazon.com/codesuite/codebuild/151559006927/projects/pn-RunFeE2eTestsCodeBuildProject/build/pn-RunFeE2eTestsCodeBuildProject%3Acd58c55f-9efe-400b-840c-408316b44ac4?region=eu-south-1
